### PR TITLE
Button component.

### PR DESCRIPTION
**Pull request by mahima-vs**
This PR containing code for generating a UI for button component.

### CODE REVIEW

File: examples/App.js [hunk 1]
```diff
@@ -5,41 +5,81 @@
...
+} from 'react-native/Libraries/NewAppScreen';
 
 import React from 'react';
-import {View} from 'react-native';
-import {get} from 'lodash';
-import UiBuilder from 'react-native-ui-builder';
+import { View, StyleSheet, Text } from 'react-native';
+// import { get } from 'lodash';
+// import UiBuilder from 'react-native-ui-builder';
+import UiBuilder from "./src/index";
```
Comment (pulkitsharma1995) [inline examples/App.js:16]: Why did you change the path to "./src/index". Since we are using a npm package.

Comment (mahima-vs) [reply examples/App.js]: This is for running locally, will use npm package later.
